cmd/probe/eventhandler.go

```diff
@@ -87,6 +109,7 @@ func (pe *ProbeEvent) deleteDiskEvent(msg controller.EventMessage) {
 // used for initial setup and when any uid mismatch or error occurred.
 func (pe *ProbeEvent) initOrErrorEvent() {
 	udevProbe := newUdevProbe(pe.Controller)
+	defer udevProbe.free()
```

Comment:
There is udevEnumerate pointer inside udevProbe which is allocated using "C" memory allocation API, we need to free that as well (I think so). Need something like "defer udevProbe.udevEnumerate.free()"

Reply:
Yes, there is `udevEnumerate` pointer inside `udevProbe`. But the [`free()`](https://github.com/openebs/node-disk-manager/pull/197/files#diff-95bc73aac6b99eb866e68f9112b105d3R210) of `udevProbe` struct handles all its memory allocations. Therefore we don't need to explicitly free the allocations.